core/src/main/java/com/linecorp/armeria/client/AbstractRuleBuilder.java

```diff
@@ -29,21 +28,19 @@
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Sets;
 
-import com.linecorp.armeria.client.UnprocessedRequestException;
+import com.linecorp.armeria.client.retry.RetryRule;
+import com.linecorp.armeria.client.retry.RetryRuleWithContent;
 import com.linecorp.armeria.common.HttpStatus;
 import com.linecorp.armeria.common.HttpStatusClass;
 import com.linecorp.armeria.common.RequestHeaders;
 import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.common.util.UnstableApi;
 
 /**
- * An abstract builder class which creates a {@link RetryRule} or a {@link RetryRuleWithContent}.
+ * A skeletal builder implementation for {@link RetryRule} and {@link RetryRuleWithContent}.
```

Comment:
I realized this class can be used for building `CircuitBreakerRule(WithContent)`.